Makefile.envs

```diff
@@ -165,7 +166,9 @@ export MAIN_MODULE_CFLAGS= $(CFLAGS_BASE) \
 	-Werror=int-conversion \
 	-Werror=incompatible-pointer-types \
 	-Werror=unused-result \
+	-mreference-types \
 	-I$(PYTHONINCLUDE) \
+	-I$(PYTHONINCLUDE)/.. \
```

Comment:
Maybe we need to find a better place to install header files later... (No action required for this PR)

Reply:
I think we can remove this once we get rid of `src/core/hiwire.h` which shadows the real `hiwire.h`. For this pr I tried not to change files outside of `hiwire.*`.